websites/X/Xero.gg/presence.ts

```diff
@@ -96,6 +96,17 @@ presence.on("UpdateData", async () => {
   } else if (document.location.pathname.includes("/download")) {
     presenceData.details = "Viewing a page:";
     presenceData.state = "Download";
+  } else if (document.location.pathname.includes("/challenge")) {
+    presenceData.details = "Viewing a page:";
+    presenceData.state = "Challenges";
+    try {
+      const challenges_title = document.querySelector(
+        "#settings-data-container > div.xero-header-standalone"
+      ).textContent;
+      presenceData.state += ` (${challenges_title})`;
+    } catch {
+      //Catch nothing
+    }
```

Comment:
If you use try function, why you don't catch anything? Just use something like this:
```javascript
const f = hh() ? jj() : ""
```